language/stdlib/modules/libra_system.mvir

```diff
@@ -17,6 +17,12 @@ module LibraSystem {
         new_validator_set: Vector.T<Self.ValidatorInfo>,
     }
 
+    struct NewBlockEvent {
+        height: u64,
+        // On-chain time during the block at the given height
```

Comment:
this comment is really hard for me to red and understand. What does it mean exactly?